ipam/pool.go

```diff
@@ -379,14 +386,34 @@ func (as *addressSpace) releasePool(poolId string) error {
 //
 // AddressPool
 //
-
-// Returns if an address pool is currently in use.
+// Returns AddressPoolInfo with Ip address statistics
 func (ap *addressPool) getInfo() *AddressPoolInfo {
+
+	var available, capacity int = 0, 0
+	var unhealthyAddr []string
+
+	for _, ar := range ap.Addresses {
+		if !ar.InUse {
+			available += 1
+		}
+		if ar.Unhealthy {
+			addr := net.IPNet{
+				IP:   ar.Addr,
+				Mask: ap.Subnet.Mask,
+			}
+			unhealthyAddr = append(unhealthyAddr, addr.String())
+		}
```

Comment:
Not sure why we are creating another IPNet here. Shouldn't this block just be unhealthyAddrs = append(unhealthyAddrs, ar.Addr)?